src/v2/styles/AttachmentList/AttachmentList-layout.scss

```diff
@@ -6,6 +6,12 @@
 
   /* The height of GIFs */
   --str-chat__gif-height: calc(var(--str-chat__spacing-px) * 200);
+
+  /* The width of the amplitude bar of the voice recording wave data */
+  --str-chat__voice-recording-amplitude-bar-width: 2px;
+
+  /* The gap between amplitudes of the wave data of a voice recording */
+  --str-chat__voice-recording-amplitude-bar-gap-width: var(--str-chat__spacing-px);
```

Comment:
Not a comment on this particular PR, but really, how is `var(--str-chat__spacing-px)` useful? 😒 Just `1px` is totally fine IMO.

Reply:
It was created after tailwind's spacing configuration: https://tailwindcss.com/docs/theme#spacing - but I also don't really see it's usefulness, nor do we use it 100% of the time